src/components/Work/Work.tsx

```diff
@@ -120,6 +128,42 @@ const WorkDetail: React.FC<{ workResult: WorkResult; backUrl?: string }> = (
               </Box>
             </>
           )}
+          {work.inCollections && work.inCollections.length > 0 && (
+            <Card
+              imageProps={{
+                alt: "Collection cover image",
```

Comment:
not sure if there is an opportunity here for the backend to send more descriptive alt text for the image, but if so that would be cool